manywheel/build.sh

```diff
@@ -183,6 +187,22 @@ DEPS_SONAME=(
     "libnvrtc-builtins.so"
     "libgomp.so.1"
 )
+elif [[ $CUDA_VERSION == "11.1" ]]; then
+DEPS_LIST=(
```

Comment:
Just curious, how is this list procured?

Reply:
it's just copy/pasted each time :/